README.md

```diff
@@ -39,6 +41,42 @@ Examples
 For now, [refer to the test cases](https://github.com/sayanarijit/expandvars/blob/master/tests) to see how it behaves.
 
 
+TIPs
+----
+
+### nounset=True
+
+If you want to enable strict parsing by default, (similar to `set -u` / `set -o nounset` in bash), pass `nounset=True`.
+
+```python
+# All the variables must be defined.
+expandvars("$VAR1:${VAR2}:$VAR3", nounset=True)
+
+# Raises UnboundVariable error.
+```
+
+> NOTE: Another way is to use the `${VAR?}` or `${VAR:?}` syntax. See the examples in tests.
+
+### export RECOVER_NULL="foo"
+
+If you want to temporarily disable strict parsing both for `nounset=True` and the `${VAR:?}` syntax, set environment variable `RECOVER_NULL=somevalue`.
+
+```bash
+export RECOVER_NULL=foo
```

Comment:
Doing this with an environment variable seems very dangerous. If you have a pipeline that involves your own tool but also someone else's tool that happens to use expandvars, then this other tool can start behaving wrong.

Reply:
Hi. Valid point too. We shouldn't use `export`. But this feature comes in handy so often that I'd still keep it.
I have added WARNINGs to remind the consequences of using `export`.

Reply:
The problem is that some of the people who would like to see the warning won't be able to see it. If Alice and Bob each create a tool, and I put them both in my stack, then Bob's tool can break when Alice's tool has a script that says "RECOVER_ALL=foo". I'm just an unsuspecting user who would be bitten by this (I probably don't even know that the tools I am relying on use expandvars internally).

Reply:
Sorry I misspelled it in the first PR (corrected it now). The name is actually `EXPANDVARS_RECOVER_NULL`. I doubt someone will use this name unintentionally.